blocks/noDemoCatalog.test.js

```
@@ -0,0 +1,19 @@
+const {test} = require('node:test');
+const assert = require('node:assert/strict');
+const fs = require('fs');
+const path = require('path');
+
+const read = (rel) => fs.readFileSync(path.join(__dirname, rel), 'utf8');
+
+test('blocks registry has no product listing widgets', () => {
+  const src = read('index.js');
+  assert.doesNotMatch(src, /ProductFilters|ProductList|ProductView|ListingSearchBar/);
```

ESLint

[error] Replace `src,·/ProductFilters|ProductList|ProductView|ListingSearchBar/` with `⏎····src,⏎····/ProductFilters|ProductList|ProductView|ListingSearchBar/,⏎··`

(prettier/prettier)